src/libs/numberFormat/index.android.js

```diff
@@ -0,0 +1,14 @@
+// we only need polyfills for Mobile.
+import '@formatjs/intl-getcanonicallocales/polyfill';
+import '@formatjs/intl-locale/polyfill';
+import '@formatjs/intl-pluralrules/polyfill';
+import '@formatjs/intl-numberformat/polyfill';
+
+// Load en Locale data
+import '@formatjs/intl-numberformat/locale-data/en';
+
+function numberFormat(locale, number, options) {
+    return new Intl.NumberFormat(locale, options).format(number);
+}
```

Comment:
Can you test if importing and using the function defined in `index.js` works properly please?

Reply:
Oh, I missed [this comment](https://github.com/Expensify/Expensify.cash/pull/2208#discussion_r608877402)